README.md

```diff
@@ -60,7 +60,7 @@ API: [FileInfo](http://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API
   ```
   All this info will be available in the *fileInfoJson* object that is returned as part of the promise resolution.
 
-### uploadFile(repoKey, remotefilePath, localfilePath, forceUpload)
+### uploadFile(repoKey, remotefilePath, fileToUploadPath, forceUpload, checksums)

```

Comment:
I think so. I'm not sure if I want to do that as part of this PR, but we might want to make a 2.0 version bump and do some options {} =)